api/cookie.go

```diff
@@ -0,0 +1,73 @@
+package api
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gorilla/securecookie"
+)
+
+const (
+	// cookieName is the name of the cookie where we store the user's JWT token.
+	cookieName = "SkynetJWTCookie"
+	// cookieValidity specifies for how long the cookie is valid. In seconds.
+	cookieValidity = 604800 // one week
+
+	// envCookieDomain holds the name of the environment variable for the
+	// domain name of the portal
+	envCookieDomain = "COOKIE_COMAIN"
```

Comment:
```suggestion
	envCookieDomain = "COOKIE_DOMAIN"
```

![](https://inteng-storage.s3.amazonaws.com/images/MARCH/sizes/Clippy-Clip_resize_md.jpg)

Reply:
Why have env vars to store the names of other env vars? I feel like we have a lot of env vars and it's getting confusing

Reply:
Ha ha ha, Clippy!